xbmc/GUIPassword.h

```diff
@@ -64,20 +65,24 @@ class CGUIPassword : public ISettingCallback
   void LockSources(bool lock);
   void RemoveSourceLocks();
   bool IsDatabasePathUnlocked(const std::string& strPath, VECSOURCES& vecSources);
-  /*! \brief Tests if the user is allowed to access the path by looking up the matching Mediasource
-   \param strPath The folder path to access
-   \param strType The type of share being accessed, e.g. "music", "video", etc. See CSettings::UpdateSources()
-   \return If access is granted, returns \e true
-   */
-  bool IsMediaPathUnlocked(const std::string& strPath, const std::string& strType);
+
+  std::string getMediasourcePath() { return strMediasourcePath; }
+  void setMediasourcePath(std::string tmpMediasourcePath) { strMediasourcePath = tmpMediasourcePath; }
```

Comment:
Function should start with an uppercase letter

Reply:
uppercase, ok, the body should go into .cpp or can it stay in .h?

Reply:
The body is fine. Simple get and set for class members is common in headers.

Reply:
param `const std::string&`, please
metod name first letter upper case, please
line no longer than 100 chars, please